app-text/unrtf/unrtf-0.21.9.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="Rich Text Format command line converter"
+DESCRIPTION="UnRTF is a command-line program written in C which converts \
+documents in Rich Text Format (.rtf) to HTML, LaTeX, troff macros, and RTF \
+itself."
+HOMEPAGE="https://www.gnu.org/software/unrtf/"
+COPYRIGHT="2001-2015 Free Software Foundation"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://www.gnu.org/software/unrtf/unrtf-$portVersion.tar.gz"
+CHECKSUM_SHA256="22a37826f96d754e335fb69f8036c068c00dd01ee9edd9461a36df0085fb8ddd"
+PATCHES="unrtf-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	unrtf$secondaryArchSuffix = $portVersion
+	cmd:unrtf = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libiconv$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir\
```

Comment:
... and insert these 2 line in BUILD before calling runConfigure:
```
	aclocal
	automake --add-missing
```
then the recipe builds fine ;-)